app/routes/pages/components/Sidebar.css

```diff
@@ -100,7 +100,7 @@
   .sidebar {
     width: 100%;
     max-width: 100%;
-    background-color: #f6f6f6;
+    background-color: var(--color-white);
```

Comment:
Could we maybe do `var(--lego-card-color)` so that it matches our other components better? 😄 

It's the same on light mode, but slightly different on dark mode.

Reply:
Fixed!